src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ReceivedZipFileRepository.java

```diff
@@ -19,7 +19,8 @@
             + "  process_events.createdat AS processingStartedEventDate, "
             + "  scannable_items.documentControlNumber AS scannableItemDCN, "
             + "  payments.documentControlNumber AS paymentDCN, "
-            + "  envelopes.rescanFor "
+            + "  envelopes.rescanFor, "
+            + "  Cast(envelopes.id as varchar) as envelopeId " // null means no envelope
```

Comment:
why casting to varchar here but then use UUID in java model?

Reply:
to map to UUID, it needs.